plugins/nemo-agents/README.md

```
@@ -227,6 +228,269 @@ suggesting recovery options instead of an opaque subprocess error.
+
+Render-only (inspect or edit the Dockerfile before building):
+
+```bash
+nemo agents package \
+    --agent examples/react-agent.yml \
+    --nat-version 1.5.0 \
+    --no-build
+```
+
+Output:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)